mimic/model/identity.py

```diff
@@ -0,0 +1,22 @@
+"""
+Models relating to identity.
+"""
+from attr import attributes, attr, validators
```

Comment:
If you've looked at the ticket before now, I just added all of the places where it would need to get changed.